cmake/mcuboot.cmake

```diff
@@ -26,29 +32,20 @@ function(zephyr_mcuboot_tasks)
     return()
   endif()
 
-  if(NOT WEST)
-    # This feature requires west.
-    message(FATAL_ERROR "Can't sign images for MCUboot: west not found. To fix, install west and ensure it's on PATH.")
-  endif()
-
-  if(NOT IS_ABSOLUTE "${keyfile}")
-    # Relative paths are relative to 'west topdir'.
-    set(keyfile "${WEST_TOPDIR}/${keyfile}")
-    set(keyfile_relative TRUE)
-  else()
-    set(keyfile_relative FALSE)
-  endif()
+foreach(file keyfile keyfile_enc)
```

Comment:
@tejlmand I did an attempt to write it the way you suggested, please have a look.
I decided to keep the extra check at line 29 to cover the case where `MCUBOOT_ENCRYPTION_KEY_FILE` is set but `MCUBOOT_SIGNATURE_KEY_FILE` is not set.